### ISSUE TITLE
SCIM Notification does not contains anymore user info

### ISSUE THREAD

**Issue by francesco-scullino** [simpleidserver/SimpleIdServer#749]
Hello,
times ago the notification about user changes was containing the user info. Now it seems it contains just the location to get them

```
{
  "id": "e84c8c0f-7d22-4bd2-b17e-4d63c45ba355",
  "entitlements": [
    {
      "value": "4382a902-82c5-4567-9360-acf4dd1136d6",
      "type": "Entitlement",
      "display": "e-markets"
    }
  ],
  "emails": [],
  "groups": [],
  "meta": {
    "resourceType": "User",
    "created": "2024-05-21T16:02:10.291Z",
    "lastModified": "2024-05-21T16:02:19.6271811Z",
    "version": 1,
    "location": "http://localhost:5289/Users/e84c8c0f-7d22-4bd2-b17e-4d63c45ba355"
  },
  "schemas": [
    "urn:ietf:params:scim:schemas:core:2.0:User"
  ]
}
```

but I'm in a scenario that does not allow my app to call this endpoint. Is there a flag that allow me to get all the data in the message payload? It would be good to use the **IsFullRepresentationReturned** flag also for notifications.

Thanks

**Comment by simpleidserver** [comment]
Hello,

Indeed, there is an issue in the current implementation. 
I made some changes to return the full representation in the Integration Event when the property is set to true.

Best regards,

SID

**Comment by francesco-scullino** [comment]
Hello and thanks for the answer.
Is there a release date for the nuget package of the Release 5.0.0?

Thanks
Francesco

**Comment by simpleidserver** [comment]
Hello,

The release of version 5.0.0 will be published next week. In the meantime, you can use the release candidate "5.0.0-rc1".

https://www.nuget.org/packages/SimpleIdServer.Scim/5.0.0-rc1

Kind regards,

SID

**Comment by francesco-scullino** [comment]
Hello,
thanks for you quick fix. Anyway I tested the new nuget package but the swagger is not rendered and the endpoints are missing, without any error in the console.
![image](https://github.com/simpleidserver/SimpleIdServer/assets/1720798/9cea36e0-fba5-45e5-8fa7-5be66f6318a1)

I just updated the library and the .NET version
![image](https://github.com/simpleidserver/SimpleIdServer/assets/1720798/cd80d6a9-bcee-4fd5-a47b-be9d40eded61)

I analyzed deeply your [Program.cs](https://github.com/simpleidserver/SimpleIdServer/blob/master/src/Scim/SimpleIdServer.Scim.Startup/Program.cs) but I cannot find the root cause
Could you please add some release notes about the new startup configuration?

Thanks

**Comment by simpleidserver** [comment]
@francesco-scullino : Unfortunately, there is a small regression in the release 5.0.0 of the SimpleIdServer.Scim project. 
I have published a new version, 5.0.1-rc1, to fix this problem. You can download it here:  https://www.nuget.org/packages/SimpleIdServer.Scim/5.0.1-rc1
